[2025-03-06T11:02:44.920+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-06T11:02:44.949+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline_demo.training_basic_classifier manual__2025-03-06T11:02:18.688108+00:00 [queued]>
[2025-03-06T11:02:44.965+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline_demo.training_basic_classifier manual__2025-03-06T11:02:18.688108+00:00 [queued]>
[2025-03-06T11:02:44.966+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-06T11:02:44.988+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): training_basic_classifier> on 2025-03-06 11:02:18.688108+00:00
[2025-03-06T11:02:45.007+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline_demo', 'training_basic_classifier', 'manual__2025-03-06T11:02:18.688108+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmp69cft9r7']
[2025-03-06T11:02:45.006+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=139) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-06T11:02:45.009+0000] {standard_task_runner.py:105} INFO - Job 77: Subtask training_basic_classifier
[2025-03-06T11:02:45.010+0000] {standard_task_runner.py:72} INFO - Started process 152 to run task
[2025-03-06T11:02:45.070+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline_demo.training_basic_classifier manual__2025-03-06T11:02:18.688108+00:00 [running]> on host 2130229946ac
[2025-03-06T11:02:45.171+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ml_pipeline_demo' AIRFLOW_CTX_TASK_ID='training_basic_classifier' AIRFLOW_CTX_EXECUTION_DATE='2025-03-06T11:02:18.688108+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-06T11:02:18.688108+00:00'
[2025-03-06T11:02:45.173+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-06T11:02:45.174+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-06T11:02:45.175+0000] {logging_mixin.py:190} INFO - Current task name:training_basic_classifier state:running start_date:2025-03-06 11:02:44.950705+00:00
[2025-03-06T11:02:45.176+0000] {logging_mixin.py:190} INFO - Dag name:ml_pipeline_demo and current dag run status:running
[2025-03-06T11:02:45.177+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-06T11:02:45.197+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/sklearn/utils/validation.py:1408: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)

[2025-03-06T11:02:45.367+0000] {logging_mixin.py:190} INFO - ML model training done and saved to /mlmodel.pkl
[2025-03-06T11:02:45.367+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-06T11:02:45.385+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-06T11:02:45.386+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ml_pipeline_demo, task_id=training_basic_classifier, run_id=manual__2025-03-06T11:02:18.688108+00:00, execution_date=20250306T110218, start_date=20250306T110244, end_date=20250306T110245
[2025-03-06T11:02:45.424+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-06T11:02:45.427+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-06T11:02:45.429+0000] {logging_mixin.py:190} INFO - Dag name:ml_pipeline_demo queued_at:2025-03-06 11:02:18.710437+00:00
[2025-03-06T11:02:45.430+0000] {logging_mixin.py:190} INFO - Task hostname:2130229946ac operator:PythonOperator
[2025-03-06T11:02:45.479+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-06T11:02:45.512+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-06T11:02:45.514+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
